File: labs/lab2.1/csharp/Program.cs
// 3rd variant
// Описати класи, об’єкт
// похідного класу;
// вивести дані об’єкта;
// обчислити і вивести площу
// та периметр
static class Porgram {
    static void Main() {
        var triangle = new Triangle([0,0],[0.5,1],[3,0]);
        Console.WriteLine(triangle);
        Console.WriteLine($"Perimeter: {triangle.GetPerimeter()}, area: {triangle.GetArea()}");
    }
}

File: labs/lab2.1/cpp/Program.cpp
#include "iostream"
#include "Triangle.cpp"

int main() {
    vector<Vertex> vertices = {Vertex(0,0),Vertex(2,1),Vertex(3,-2)};
    Triangle triangle1(vertices);
    cout << triangle1.ToString() << '\n';
    cout << triangle1.GetArea() << ' ' << triangle1.GetPerimeter() << '\n';

    Triangle copiedTriangle(triangle1);
    cout << copiedTriangle.ToString() << '\n';
    return 0;
}